Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/03-TestingPresentation/02-03-TestingPresentation.tutorial

```diff
@@ -308,6 +308,9 @@
       @Step {
         Make use of the new `deleteConfirmation` static alert function in the `ContactsFeature` 
         reducer, rather than building `AlertState` from scratch.
+
+        Also to further assert that the `confirmDeletion` action was received, 
+        annotate `ContactsFeature.Action.Alert` with the `@CasePathable` macro. 
```

Comment:
📓 Add a comment in the tutorial to emphasize that I've also added the macro here for future code (and to avoid having to add more steps later on for something already explain before)